crates/krata/proto/krata/v1/control.proto

```diff
@@ -35,6 +35,8 @@ service ControlService {
     rpc ReadZoneMetrics(ReadZoneMetricsRequest) returns (ReadZoneMetricsReply);
 
     rpc WatchEvents(WatchEventsRequest) returns (stream WatchEventsReply);
+
+    rpc ReadHypervisorConsoleRing(ReadHypervisorConsoleRingRequest) returns (ReadHypervisorConsoleRingReply);
```

Comment:
This all looks very good but I want to know if it would be feasible to rename this RPC to avoid the use of "Console Ring" terminal and perhaps also reconsider "clear" in the request below. I say this only because we have both signaled interest in trying to make the API hypervisor agnostic such that we can go any direction we would like in the future.

My personal proposal would be to remove `clear` and change the RPC to `ReadHypervisorConsole(ReadHypervisorConsoleRequest) returns (ReadHypervisorConsoleReply)`

Reply:
Done in 290b8fa